README.md

```diff
@@ -12,7 +19,10 @@ OpenSearch Go Client
 
 ## Welcome!
 
-**opensearch-go** is [a community-driven, open source fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/) of go-elasticsearch licensed under the [Apache v2.0 License](LICENSE.txt). For more information, see [opensearch.org](https://opensearch.org/).
+**opensearch-go** is
+[a community-driven, open source fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)
+of go-elasticsearch licensed under the [Apache v2.0 License](LICENSE.txt). For
+more information, see [opensearch.org](https://opensearch.org/).
```

Comment:
This seems unrelated, and can we please not fold markdown?

https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html